setup.py

```diff
@@ -50,7 +50,7 @@ def find_version(*file_paths):
                     "graphviz",
                     "jsonschema",
                     "pydot",
-                    "tensorboardX",
+                    "tensorboard",
```

Comment:
Now that we have removed the external `tensorboardX` dependency and no repository code file is using it, why do we introduce another unused dependency?

Reply:
pytorch.utils.tensorboard utilizes tensorboard package

Reply:
Ok, please set a specific version via `==`